services.yaml

```diff
@@ -4784,6 +4830,60 @@ definitions:
         items:
           type: string
           example: MyApp
+  Agent:
```

Comment:
things i want to add here:
collector_id 
type (to know it's a JVM or .NET agent)

Reply:
will add